hail/python/hail/docs/change_log.md

```diff
@@ -1,15 +1,40 @@
 # Change Log
 
-## 0.2.22
+## Version 0.2.23
+
+Released 2019-09-23
+
+### `hailctl dataproc`
+- (hail#7087) Added back progress bar to notebooks, with links to the correct Spark UI url.
+- (hail#7104) Increased disk requested when using `--vep` to address the "colony collapse" cluster error mode.
+
+### Bug fixes
+- (hail#7066) Fixed generated code when methods from multiple reference genomes appear together.
+- (hail#7077) Fixed crash in `hl.agg.group_by`.
+ 
+### New features
+- (hail#7009) Introduced analysis pass in Python that mostly obviates the `hl.bind` and `hl.rbind` operators; idiomatic Python that generates Hail expressions will perform much better.
+- (hail#7076) Improved memory management in generated code, add additional log statements about allocated memory to improve debugging.
+- (hail#7085) Warn only once about schema mismatches during JSON import (used in VEP, Nirvana, and sometimes `import_table`.
+- (hail#7106) `hl.agg.call_stats` can now accept a number of alleles for its `alleles` parameter, useful when dealing with biallelic calls without the alleles array at hand.
+ 
+### Performance
+- (hail#7086) Improved performance of JSON import.
+- (hail#6981) Improved performance of Hail min/max/mean operators. Improved performance of `split_multi_hts` by an additional 33%.
+- (hail#7082)(hail#7096)(#7098) Improved performance of large pipelines involving many `annotate` calls.
```

Comment:
does that guy need a `hail` before the `#`?

Reply:
yep

Reply:
done